src/fty_proto.xml

```diff
@@ -52,19 +52,30 @@
     id = "1"
     >
     <field name = "type" type = "string">
-        Type of metric send (temperature, humidity, power.load, ...)
+        Metric name, e.g.: "temperature", "humidity", "power.load", ... 
     </field>
-    <field name = "element_src" type = "string">
-        Name of source element to which metrics are bound to
+
+    <field name = "name" type = "string">
+        Name of asset where metric is produced.
     </field>
+
     <field name = "value" type = "string">
-        Value of metric as plain string
+        Metric value, e.g.: "25.323" or "900".
     </field>
+
     <field name = "unit" type = "string">
-        Unit of metric (i.e. C, F or K for temperature)
+        Metric unit , .e.g.: "C" or "F" for temperature.
     </field>
+
     <field name = "ttl" type = "number" size = "4">
-        Metric time to live seconds (i.e. How long is the metric valid - At the latest how long from now should i get a new one)
+        Time to live in seconds.
+
+        Value defines duration of validity (i.e. how long is metric valid, when
```

Comment:
Note that generally in IT, the TTL != polling rate, although they are often connected to account for intermittent communications issues - usually TTL=3*POLLRATE in routing and similar time-limited protocols, and may be a much bigger ratio in e.g. DNS (poll every few minutes, but trust old data for days if the origin DNS server has problems).

In short: it should not be 1:1 (refresh when old data expires) - this mode will always break :)